content/pages/3.contribute/2.build/1.build-release.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Any reason to add a blank line at the beginning of this file ?

Reply:
This is my coding style both for content and code, always add a blank line at the beginning and ending of files to ease selection and visualization.